public/pages/IndexDetail/containers/IndexDetail/IndexDetail.tsx

```diff
@@ -83,7 +83,7 @@ export default function IndexDetail(props: IndexDetailModalProps) {
     const result = await services.indexService.getIndices({
       terms: index,
       from: 0,
-      size: 10,
+      size: 999,
```

Comment:
any reason changed it to 999?

Reply:
The getIndices API fetch indexes by using "like", and if there are more than 10 indexes matched, the target Index can not be found.

Reply:
But good point, I will change to add exactSearch on getIndices method and lower the size to 10.